core/src/main/scala-2.13+/cats/data/ScalaVersionSpecificPackage.scala

```diff
@@ -1,7 +1,19 @@
 package cats
-
 package data
 
+import kernel.compat.scalaVersionSpecific._
+
 abstract private[data] class ScalaVersionSpecificPackage {
   type NonEmptyLazyList[+A] = NonEmptyLazyList.Type[A]
+  @deprecated("2.0.0-RC2", "Use NonEmptyLazyList")
+  type NonEmptyStream[A] = OneAnd[Stream, A]
+
+  @deprecated("2.0.0-RC2", "Use NonEmptyLazyList")
+  def NonEmptyStream[A](head: A, tail: Stream[A]): NonEmptyStream[A] =
```

Comment:
Note that we can't have the default argument here that we have on 2.11 and 2.12, since it results in a deprecation warning (which is clearly a compiler bug).